vllm/v1/metrics/stats.py

```diff
@@ -48,6 +52,8 @@ def update_from_output(self, output: "EngineCoreOutput",
             return
 
         num_new_generation_tokens = len(output.new_token_ids)
+        now = time.time()
```

Comment:
I have seen mixed use of `time.time()` and `time.monotonic()` - I think we should standardize on one or the other so we don't have a footgun

Reply:
Yes!

I had initially changed to `time.monotonic()` but I backed it out because it looks like `arrival_time` being wall-clock time is part of the public API, so not something I wanted to rush into.

I'll make sure to do it or capture it as a TODO/FIXME before I wrap this up :+1: